app/v2/models/response/retrieveHistoricNonFhlUkPiePeriodSummary/PeriodIncome.scala

```diff
@@ -20,10 +20,10 @@ import play.api.libs.functional.syntax._
 import play.api.libs.json.{ JsPath, Json, OWrites, Reads }
 
 case class PeriodIncome(periodAmount: Option[BigDecimal],
-                        taxDeducted: Option[BigDecimal],
                         premiumsOfLeaseGrant: Option[BigDecimal],
                         reversePremiums: Option[BigDecimal],
                         otherIncome: Option[BigDecimal],
+                        taxDeducted: Option[BigDecimal],
```

Comment:
The reads don't match the case class, taxDeducted is read into premiumsOfLeaseGrant. Needs to be fixed to have correct reads and order of case clas and also tests updated